marketplace_builder/custom_model_types/car.yml

```
@@ -0,0 +1,5 @@
+name: Car
+custom_attributes:
```

you can use `properties:`

---

you can use suggest mode to make CR process faster and easier @Slashek  :)

```suggestion
properties:
```